app/globals.css

```
@@ -14,46 +14,240 @@
+  box-shadow: 0 -4px 20px rgba(0, 0, 0, 0.15);
+  backdrop-filter: blur(12px);
+  -webkit-backdrop-filter: blur(12px);
+  /* Ensure bottom spacing */
+  margin-bottom: 0.5rem;
+  /* Prevent container from growing too large - limit to 25% of viewport */
+  max-height: 25vh;
+  overflow: visible;
+}
+
+/* Import mobile chat styles */
+@import url('./chat/styles/mobile-chat.css');
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)